qiskit/quantum_info/states/stabilizerstate.py

```diff
@@ -231,10 +235,10 @@ def expectation_value(self, oper, qargs=None):
         # Check if there is a stabilizer that anti-commutes with an odd number of qubits
         # If so the expectation value is 0
         for p in range(num_qubits):
-            stab = self.clifford.stabilizer
+            stab = self.clifford.paulis[num_qubits : 2 * num_qubits]
```

Comment:
I think that `stabilizer` is more clear

Reply:
Thank you. How about this? 
https://github.com/Qiskit/qiskit-terra/pull/7269/commits/4ddbb2a4224b0ae4eacf234a96bbbad107ead101#diff-bd8577b12bcae2760cf0d430eae01541d035c76d9d1f8fff1d04f89e7a8b4049R238

Reply:
OK, but still more complicated then just `self.clifford.stabilizer`

Reply:
I see. I made an alias for this.
https://github.com/Qiskit/qiskit-terra/pull/7269/commits/7f29d21ee836cf5308362cb600e6252c534ef7d5#diff-bd8577b12bcae2760cf0d430eae01541d035c76d9d1f8fff1d04f89e7a8b4049R236

Reply:
Thanks - I think this is better.